lib/class-wp-rest-menus-controller.php

```diff
@@ -146,6 +155,25 @@ protected function check_assign_locations_permission( $request ) {
 		return true;
 	}
 
+	/**
+	 * Checks whether current user can set auto add pages.
+	 *
+	 * @param WP_REST_Request $request The request object with post and locations data.
+	 *
+	 * @return bool|WP_Error Whether the current user can assign the provided terms.
+	 */
+	protected function check_set_auto_add_permission( $request ) {
```

Comment:
Added it's own permissions here. Not sure if this is the right call. Thoughts @TimothyBJacobs

Reply:
I think that's fine we follow the pattern elsewhere.